tests/run_selective.py

```
@@ -57,18 +65,18 @@
 	args = parser.parse_args()
 	try:
 		_bar = str("-" * 20)
-		if (sys.stdout.isatty()):
-			print(f"{_bar}START{_bar}", file=sys.stdout)
+		logger = logging.getLogger(__module__)
+		logger.info(f"{_bar}GROUP{_bar}")  # skipcq PYL-W1203 - test code ok
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)